index.html

```diff
@@ -0,0 +1,488 @@
+                    </li>
+                    <li class="flex">
+                      <img src="./static/imgs/css.png" alt="css" /><span
+                        >CSS</span
+                      >
+                    </li>
+                  </ul>
+                </li>
+
+                <li>
+                  Frameworks<span class="icon"
+                    ><img src="./static/imgs/right.png" alt="right.png"
+                  /></span>
+                </li>
+
+                <li>
+                  Skills<span class="icon"
+                    ><img src="./static/imgs/right.png" alt="right.png"
+                  /></span>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+    <!-- end sectionsix -->
+    <!-- start footer -->
+    <div class="white" id="contact">
+      <footer class="footer" id="footer">
+        <div class="container">
+          <div class="footer__container__parfoot">
+            <h3>Contact me</h3>
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+            <form action="https://formspree.io/f/maykokoe" method="post">
+              <div class="form__control">
+                <label for="name">Name</label>
+                <input
+                  type="text"
+                  placeholder="Name"
+                  class="name"
+                  id="name"
+                  maxlength="30"
+                  required
+                  name="name"
+                />
+              </div>
+              <div class="form__control">
+                <label for="email">name@example.com</label>
+                <input
+                  type="email"
+                  placeholder="name@example.com"
+                  class="email"
+                  id="email"
+                  name="email"
+                  required
+                />
+              </div>
+              <div class="form__control">
+                <label for="textarea">Write your message here</label>
+                <textarea
+                  id=""
+                  cols="30"
+                  rows="10"
+                  class="textarea"
+                  id="textarea"
+                  placeholder="Write your message here"
+                  maxlength="500"
+                  name="textarea"
+                  required
+                ></textarea>
+              </div>
+              <p class="form__control" id="emailerror">
+                Your email is invalid, email should be in lowercase
+              </p>
+              <div class="form__control">
+                <input
+                  type="submit"
+                  placeholder=""
+                  class="submit"
+                  id="submit"
+                  value="Get in touch"
+                />
```

Comment:
[OPTIONAL] Nice work implementing the form. However, I realized that the form inputs are not cleared after submitting. Kindly make sure that the inputs are clear whenever you submit since that is the standard best practice. :+1: 

![Screenshot from 2022-10-19 13-18-52](https://user-images.githubusercontent.com/65233860/196702205-6dacd00f-a2ac-42d3-b0dd-02090941d7fc.png)

Reply:
@jerryowusu,
thanks for your good and valuable feedback,
yes you mention a very specific and important point. I well take in to account your comments in my future tasks.